pkgs/development/python-modules/pycryptodome/default.nix

```diff
@@ -9,6 +9,10 @@ buildPythonPackage rec {
     sha256 = "f1add21b6d179179b3c177c33d18a2186a09cc0d3af41ff5ed3f377360b869f2";
   };
 
+  preBuild = ''
+    export LDSHARED="$CC -shared" # Fix cross compilation, defaults to gcc instead of $CC
```

Comment:
Is this a variable python uses or this project in particular?

Reply:
It's something that `distutils` uses and it would be best to fix this in our Python abstraction if someone has time for it. I've opened #91171 and added some information there, but I didn't really dig deeper into this. For me this was only a quick fix for cross compiling another package. We can either merge this until we have a proper fix or close this PR (I'm fine with both).

Reply:
https://stackoverflow.com/questions/5986256/python-distutils-gcc-path